src/Payment.cs

```diff
@@ -43,5 +43,30 @@ public Refund Refunds
                 return refund;
             }
         }
+
+        public Transfer Transfer(Dictionary<string, object> data = null)

```

Comment:
add constructor which takes payment id and can call Transfer method, fix whereever applies